chromium_src/chrome/browser/ui/webui/webui_util.cc

```diff
@@ -13,7 +13,8 @@ namespace webui {
 
 namespace {
 constexpr char kBraveCSP[] =
```

Comment:
is this CSP being applied to just the VPN panel or other webUI pages as well?

Reply:
others too

Reply:
is it possible to do this change without adding to the CSP? am curious why it's needed

Reply:
well it's so that the pages in chrome-untrusted can load JS from chrome-untrusted://resources.

I guess I can add the hosts from the different protocols separately depending on whether the webui is chrome-untrusted:// or chrome://

Reply:
that sounds much better to me, thanks!

Reply:
I've done that now